db/migrations/20220705193207_CreateClucks.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema.createTable("clucks",table=>{
+    table.increments("id");
+    table.string('username');
+    table.text("imageUrl");
```

Comment:
Is there a reason to why you used `text` over `string`? No right or wrong answer here since you used `text` here I wanted to know if you searched up the difference between `text` and `string` from knex doc haha :D